doc/source/getting_started/docker.rst

```diff
@@ -0,0 +1,343 @@
+.. _pymapdl_docker:
+
+****************
+MAPDL and Docker
+****************
+
+You can run MAPDL within a container on any OS using Docker and
+connect to it via PyMAPDL.
+
+There are several situations in which it is advantageous to run MAPDL
+in a containerized environment such as Docker or Singularity:
+
+- Run in a consistent environment regardless of the host OS.
+- Portability and ease of install.
+- Large-scale cluster deployment using Kubernetes
+- Genuine app isolation through containerization
+
+In this case, you use your local Python installation to
+connect to this instance.
+
+.. graphviz::
+
+    digraph {
+      node [];
+      subgraph cluster_frontend {
+        label="*HOST*";
+        Python;
+      subgraph cluster_backend {
+        MAPDL;
+        label="*DOCKER*";
+        }
+      }
+      Python -> MAPDL
+      MAPDL -> Python
+    }
+
+Requirements
+============
+
+You need to have access to a Docker image with MAPDL in it.
+Visit :ref:`ref_make_container` for more information on how
+to create your own MAPDL docker image.
+
+Once you have created and uploaded your Docker image to a registry,
+you can start to pull and use the image in other devices.
+
+.. warning::
+
+   MAPDL Docker images are not allowed to be shared in
+   public or free-to-access repositories or registries.
+   Doing this you violate Ansys policy.
+
+
+
+Configure Docker to access a private GitHub registry
+----------------------------------------------------
+
+In case you have created a Docker image and upload it to a GitHub
+private repository, you must authorise your docker installation
+to access this private package by using a personal access
+token.
+Create a GitHub personal access token with ``packages read`` permissions
+according to `Creating a personal access token <gh_creating_pat_>`_
+
+Save that token to a file with:
```

Comment:
```suggestion
Save that token to a file with this command:
```
For accessibility, the Google developer documentation style guide says to introduce a code block (or list) with a full sentence.